index.html

```diff
@@ -6,14 +6,231 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" type="text/css" href="css/grid.css">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <!-- Add a link to your custom CSS here -->
 
     <title>Responsive grid project</title>
 </head>
 
 <body>
+    
+    <header>
+             <div class="container">
+                <div class="row">
+                    <div class=" top col-12">
+                        <figure class=“logo col-3”>
+                            <img src="./images/logo1.png" alt="logo" />
+                        </figure>
+                         
+                        <nav class="top-navigation col-7">
+                            <ul>
+                                <li><a href=“#” class=“col-1">Home</a></li>
+                                <li><a href=“#” class=“col-1">Shop</a></li>
+                                <li><a href=“#” class=“col-1">Contact</a></li>
+                                <li><a href=“#” class=“col-1">About</a></li>
+                                <li><a href=“#” class=“col-1">FAQ</a></li>
+                            </ul>
+                        </nav>
+                    </div>
+                </div>
+            </div>    
+    </header> 
+    
+    <main>
+        <div class="container">
+            <div class="row  title-row">
+                <div class="col-12 md-col-12">
+                <div class="main-centre">
+                         <div class="main">
+                        <h1>Bakerdays</h1>
+                        <p>Personalised Cakes Online and For All Occasions</p>
+                     <button class="shop-now">Shop Now</button>
+                    </div>
+                    
+                </div>
+            </div>
+            </div>
+        </div>
 
-    <!-- Add your markup here -->
+        
+    <section class="container">
+        
+        <h2>Cakes</h2>
+        <div class="row">
+            
+            <div class="col-12 md-col-4">
+                <div class="card"> 
+                    <img src="./images/valentine.jpeg" alt="valentine cakes">
+                    <h5>Valentine cakes</h5>
+                    <h5>Price: £50</h5>
+                </div>
+            </div>
+            <div class="col-12 md-col-4">
+                <div class="card">
+                    <img src="./images/birthday.jpeg" alt="birthday cakes">
+                    <h5>Birthday cakes</h5>
+                    <h5>Price: £35</h5>
+                </div>
+            </div>
+            <div class="col-12 md-col-4">
+                <div class="card">
+                    <img src="./images/events.jpeg" alt="event cakes">
+                    <h5>Events cake</h5>
+                    <h5>Price: £100</h5>
+                </div>
+            </div>
```

Comment:
Very good split of the columns inside the row here 👍 

Remember to close the `container` and `row` though - you're opening a new container and row on lines 83 and 84, but you haven't closed to the previous ones.